src/code/z_effect_soft_sprite_old_init.c

```diff
@@ -91,63 +91,76 @@ void EffectSsDust_Spawn(PlayState* play, u16 drawFlags, Vec3f* pos, Vec3f* veloc
 
 void func_800B0DE0(PlayState* play, Vec3f* pos, Vec3f* velocity, Vec3f* accel, Color_RGBA8* primColor,
                    Color_RGBA8* envColor, s16 scale, s16 scaleStep) {
-    EffectSsDust_Spawn(play, 0, pos, velocity, accel, primColor, envColor, scale, scaleStep, 10, 0);
+    EffectSsDust_Spawn(play, DUST_DRAWFLAG0, pos, velocity, accel, primColor, envColor, scale, scaleStep, 10,
+                       DUST_UPDATE_NORMAL);
 }
```

Comment:
Not part of this PR, but do you have any ideas how we could name these 13 variations of dust spawn functions? `func_800B0DE0` to `func_800B12F0`?

Reply:
Not yet, it is going to need to come with the further doc pass of dust later. Mainly need to figure out what each drawflag actually does, then decided names based on that and the other slight differences.